### PR TITLE
Add blog post for Fluent Bit

### PR DESCRIPTION
**Pull request by brijos**
Signed-off-by: Joshua Bright <jdbright@amazon.com>

### Description
[Added Fluent Bit 1.9 blog post]
 
### Issues Resolved
[[List any issues this PR will resolve](https://github.com/opensearch-project/project-website/issues/733)]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2022-03-24-getting-started-with-fluent-bit-and-opensearch.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+     Name opensearch
+     Match *
+     Host 192.168.2.3
+     Port 9200
+     Index my_index
+     Type my_type
+ ```
+ In this scenario, we use the CPU log metric input plugin to retrieve CPU metrics and send them to OpenSearch.
+
+ Within a few minutes we should be able to see the CPU metrics under the my_index index within OpenSearch.
```
Comment (stockholmux) [inline _posts/2022-03-24-getting-started-with-fluent-bit-and-opensearch.md:60]: "my_index" should be in backticks

**Review by stockholmux — CHANGES_REQUESTED** [review]
Content seems fine. Spacing is causing some rendering chaos.

File: _posts/2022-03-24-getting-started-with-fluent-bit-and-opensearch.md (hunk 1 above)
Comment (brijos) [reply _posts/2022-03-24-getting-started-with-fluent-bit-and-opensearch.md:60]: Corrected!

File: _posts/2022-03-24-getting-started-with-fluent-bit-and-opensearch.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+---
+
+ We recently announced the release of Fluent Bit 1.9, and while there are a number of new features and enhancements to its already impressive speed, scale, and efficiency, one feature we are really excited about is the OpenSearch plugin for Fluent Bit. Fluent Bit is an Apache 2.0 open source lightweight log and metric processor that can gather data from many sources, while the OpenSearch project is a community-driven open-source search and analytics suite derived from Elasticsearch 7.10.2 and Kibana 7.10.2.
+
+ In this Getting Started Guide we cover 1) the new plugin with integration to OpenSearch, 2) a deployment scenario on top of Kubernetes, and 3) a bonus using some of Fluent Bit’s new 1.9 features.
```
[the remark this answers is not in the document]
Comment (brijos) [reply _posts/2022-03-24-getting-started-with-fluent-bit-and-opensearch.md:15]: For the sake of consistency, I'll adjust it.  Thanks!

**Comment by stockholmux** [comment]
FWIW, we just need one more set of eyes (in process) before we can push this live.

**Pull request merged**